addon.xml

```diff
@@ -1,15 +1,11 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <addon id="metadata.themoviedb.org.python"
        name="The Movie Database Python"
-       version="1.2.1"
+       version="1.2.2"
```

Comment:
I don't think this is a bugfix version :D

Reply:
Fair enough. I've always thought of the versioning as major.minor.maintenance and this feels like maintenance. But if 1.3.0 would be better, that's fine with me.